docs/getting-started.md

```diff
@@ -3,73 +3,97 @@
+- Download and install Elasticsearch =< 5.x (see [installation guide](https://www.elastic.co/guide/en/elasticsearch/reference/5.6/install-elasticsearch.html), Elasticsearch 6.0 is not supported by the plugin)
+- Download and install Couchbase Server >= 3.0 (see [installation guide](https://www.couchbase.com/downloads))
+
+Verify the Elasticsearch cluster is up and running (the default port is `9200`).
+
+```bash
+$ curl localhost:9200
+
+{
+  "name" : "K3RqW4F",
+  "cluster_name" : "elasticsearch",
+  "cluster_uuid" : "Bw-Ta0wDTcekzQIhXZHGkg",
+  "version" : {
+    "number" : "5.6.5",
+    "build_hash" : "6a37571",
+    "build_date" : "2017-12-04T07:50:10.466Z",
+    "build_snapshot" : false,
+    "lucene_version" : "6.6.1"
+  },
+  "tagline" : "You Know, for Search"
+}
+```
+
+Verify that Couchbase Server is running.
+
+```bash
+$ curl localhost:8092
+
+{"couchdb":"Welcome","version":"v4.5.1-60-g3cf258d","couchbase":"5.0.2-5506-community"}
+```
 
 ## Installation
 
-1. Download the Elasticsearch 2.4.0 package for Ubuntu.
+Navigate to the Elasticsearch installation directory.
+
+```bash
+$ cd /usr/share/elasticsearch
+```
+
+Download and install the plug-in package.
 
-	```bash
-	$ wget https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch-2.4.6.deb
-	```
+```bash
+$ bin/elasticsearch-plugin install https://github.com/couchbaselabs/elasticsearch-transport-couchbase/releases/download/3.0.0-cypress/elasticsearch-transport-couchbase-3.0.0-cypress-es5.6.4.zip
+```
 
-2. Install the package.
+Replace the plugin URL with the one that matches your Elasticsearch version, all URLs can be found on the [releases page](https://github.com/couchbaselabs/elasticsearch-transport-couchbase/releases).
 
-	```bash
-	$ dpkg -i elasticsearch-2.4.6.deb
-	```
+When the installation is successful, the message "Installed transport-couchbase into /usr/share/elasticsearch/plugins/transport/couchbase" will be logged.
 
-3. Navigate to the Elasticsearch installation directory.
+## Configuration
 
-	```bash
-	$ cd /usr/share/elasticsearch
-	```
+Open the Elasticsearch configuration file (**/etc/elasticsearch/elasticsearch.yml**) and add the following to the end of the file.
 
-4. Download and install the plug-in package.
+```yaml
+couchbase.username: <USERNAME>
+couchbase.password: <PASSWORD>
+couchbase.maxConcurrentRequests: 1024
+```
 
-	```bash
-	$ bin/plugin install https://github.com/couchbaselabs/elasticsearch-transport-couchbase/releases/download/3.0.0-cypress/elasticsearch-transport-couchbase-3.0.0-alder-es2.4.6.zip
-	```
+The **username** and **password** credentials will be used again later when configuring Couchbase Server.
 
-When the installation is successful, the message "Installed transport-couchbase into /usr/share/elasticsearch/plugins/transport/couchbase" is logged.
+Still in the **/usr/share/elasticsearch** directory, configure the Elasticsearch plugin with the following curl command (Couchbase Server must be running).
```

Comment:
I don't think Couchbase needs to be up for this step. Do you mean, "Elasticsearch server must be running"?